x-pack/plugins/serverless_search/public/application/constants.ts

```diff
@@ -5,7 +5,15 @@
  * 2.0.
  */
 
+import { i18n } from "@kbn/i18n";
+
 export const API_KEY_PLACEHOLDER = 'your_api_key';
 export const ELASTICSEARCH_URL_PLACEHOLDER = 'https://your_deployment_url';
 export const CLOUD_ID_PLACEHOLDER = '<CLOUD_ID>';
 export const INDEX_NAME_PLACEHOLDER = 'index_name';
+export const LEARN_MORE_LABEL: string = i18n.translate(
```

Comment:
We have a separate file for translations in `common/i18n_strings.ts`, this should go there. But that file already contains a `LEARN_MORE_LABEL`, so you should just be using that instead.